### DISCUSSION TITLE
Feature request - scroll both sides

### DISCUSSION THREAD

**Discussion by cedrus-8864** [josdejong/svelte-jsoneditor#492] [Ideas]
I have 2 JSON data and they have a few different value. So it will very helpful if I can scroll both sides and compare them.
Thank you.

**Comment by josdejong** [comment]
Thanks for your input. The web application https://jsoneditoronline.org/ actually has support for comparing two documents and scrolling through the differences one by one: 

https://jsoneditoronline.org/docs/#compare

Is that what you mean?

**Comment by cedrus-8864** [comment]
I'm using that feature, too. And I love it.
I am imagining when I scroll left side, the right side is scrolled too.

**Comment by josdejong** [comment]
Ah, I get it. That is an interesting idea, thanks! I'll move this idea from the issues section to the discussions section.

**Comment by AlexRMU** [comment]
What about synchronization of selection and expand, between the same and different modes?
But it's problematic: https://github.com/josdejong/svelte-jsoneditor/issues/487#issuecomment-2629520990
